docs/interfaces/tensorboard.rst

```diff
@@ -169,20 +176,20 @@ To add TensorBoard support for models that use the :doc:`PyTorch API
        def train_batch(self, batch, epoch_idx, batch_idx):
            self.logger.writer.add_scalar("my_metric", np.random.random(), batch_idx)
 
-For a full-length example of using TensorBoard with PyTorch, see the :download:`mnist-GAN model
-</examples/gan_mnist_pytorch.tgz>`.
+For a full-length example of using TensorBoard with PyTorch, check out the :download:`mnist-GAN
+model </examples/gan_mnist_pytorch.tgz>`.
 
-**********************
- Lifecycle Management
-**********************
+**********************************
+ TensorBoard Lifecycle Management
+**********************************
 
-Determined will automatically terminate idle TensorBoard instances. A TensorBoard instance is
-considered idle if it is does not receive HTTP traffic (a TensorBoard that is still being viewed by
-a web browser will not be considered idle). By default, idle TensorBoards will be terminated after 5
-minutes; the timeout duration can be changed by editing ``tensorboard_timeout`` in the :ref:`master
-config file <master-config-reference>`.
+Determined automatically terminates idle TensorBoard instances. A TensorBoard instance is considered
+idle if it is does not receive HTTP traffic (a TensorBoard that is still being viewed by a web
```

Comment:
@MikhailKardash typo, please change the sentence to:

`A TensorBoard instance is considered idle if it does not receive HTTP traffic...`